docs/osquery/alerts-run-osquery.asciidoc

```diff
@@ -35,7 +35,7 @@ TIP: Refer to {kibana-ref}/osquery.html#osquery-prebuilt-packs-queries[prebuilt
 [role="screenshot"]
 image::images/setup-query.png[width=80%][height=80%][Shows how to set up a single query]
 
-. Click **Submit**. Queries will time out after 5 minutes if there are no responses. Otherwise, query results display within the flyout.
+. Click **Submit**. Query results will display within the flyout.
```

Comment:
I wonder if it's worth keeping the information about queries timing out. If the default is now 1 minute, we could say something like:

> Query results will display within the flyout. If there are no responses, queries will time out after one minute, unless a custom timeout period was set.

It may not be necessary, though, since we do mention in step 4 that the timeout setting's default value is 60. I'll leave it up to you :)

Reply:
yeah, I had considered this, but I think mentioning the timeout setting and default configuration in step 4 is sufficient. : )